TurkishFilter/sections/specific.txt

```diff
@@ -107,6 +107,8 @@ multiplayer.com.tr##.sidebar-inner > #text-10
 ||cloudinary.com^$domain=dizilab.pw
 csgohileleri.org##.lalulintas
 gamer.com.tr###pskinad
+politikars.com##.base-side > .banner
```

Comment:
why not `.banner-side` or `.banner`?

Reply:
don't know, `.banner-side` it seems sufficient